packages/web-components/fast-foundation/src/text-field/text-field.template.ts

```diff
@@ -2,6 +2,17 @@ import { html, ref, slotted } from "@microsoft/fast-element";
 import { endTemplate, startTemplate } from "../patterns";
 import { TextField } from "./text-field";
 
+/***
+ * A simple whitespace filter for the template
+ * @public
+ */
+export function whitespaceFilter(value, index, array) {
+    return value.nodeType === 3 &&
+        typeof value.nodeValue === "string" &&
+        !value.nodeValue.replace(/\s/g, "").length
+        ? false
+        : true;
```

Comment:
```suggestion
    return value.nodeType === Node.TEXT_NODE &&
        typeof value.nodeValue === "string" &&
        !value.trim().length
```

Reply:
I like the more clear `node.text_node`, add to a commit now.